Software/src/wizard/LightpackDiscoveryPage.cpp

```diff
@@ -57,13 +57,8 @@ void LightpackDiscoveryPage::initializePage() {
 
 		_ui->rbLightpackSelected->setChecked(true);
 
-		QString caption;
-		if (lpack->lightpacksFound() == 1)
-			caption = tr("1 Lightpack found");
-		else
-			caption = tr("%0 Lightpacks found").arg(QString::number(lpack->lightpacksFound()));
-
-		QString caption2 = tr("%0 zones are available").arg(QString::number(lpack->maxLedsCount()));
+		QString caption = tr("%n Lightpack(s) found", 0, lpack->lightpacksFound());
+		QString caption2 = tr("%n zones are available", 0, lpack->maxLedsCount());
```

Comment:
This does the opposite of what the description says, it removes a plural, right?

What exactly matters in Polish?

Reply:
It actually adds correct plural forms instead of hard coding a specific rule for English.

see: https://doc.qt.io/archives/qq/qq19-plurals.html

Reply:
I see. Sorry this was new to me. Thanks!